Well, it looks like AppVeyor has caught an incompatibility with new versions of pytest:
https://ci.appveyor.com/project/drothlis/stbt-rig/builds/35072163/job/atimtgkwgppxebqj

```
________________________ ERROR collecting test session ________________________
c:\python36-x64\lib\site-packages\pluggy\hooks.py:286: in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
c:\python36-x64\lib\site-packages\pluggy\manager.py:93: in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
c:\python36-x64\lib\site-packages\pluggy\manager.py:87: in <lambda>
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
C:\projects\stbt-rig\stbt_rig.py:1227: in pytest_collect_file
    return StbtCollector(path, parent)
c:\python36-x64\lib\site-packages\_pytest\nodes.py:95: in __call__
    warnings.warn(NODE_USE_FROM_PARENT.format(name=self.__name__), stacklevel=2)
E   pytest.PytestDeprecationWarning: Direct construction of StbtCollector has been deprecated, please use StbtCollector.from_parent.
E   See https://docs.pytest.org/en/stable/deprecations.html#node-construction-changed-to-node-from-parent for more details.
```

I'm going to stop working on this now so back over to you @wmanley.

--- a/stbt_rig.py
+++ b/stbt_rig.py
@@ -1235,6 +1235,7 @@ try:
 
 
     class StbtCollector(pytest.File):
+        # pylint: disable=abstract-method
         def collect(self):
             with open(self.fspath.strpath) as f:
                 # We implement our own parsing to avoid import stbt ImportErrors
@@ -1255,6 +1256,7 @@ try:
 
 
     class StbtRemoteTest(pytest.Item):
+        # pylint: disable=abstract-method
         def __init__(self, parent, filename, testname, line_number):
             super(StbtRemoteTest, self).__init__(testname, parent)
             self._filename = filename

--- a/stbt_rig.py
+++ b/stbt_rig.py
@@ -6,7 +6,7 @@
 For more details, and to get the latest version of this script, see
 <https://github.com/stb-tester/stbt-rig>.
 
-Copyright 2017-2019 Stb-tester.com Ltd. <support@stb-tester.com>
+Copyright 2017-2020 Stb-tester.com Ltd. <support@stb-tester.com>
 Released under the MIT license.
 """
 

--- a/stbt_rig.py
+++ b/stbt_rig.py
@@ -1223,7 +1223,13 @@ try:
 
     def pytest_collect_file(path, parent):
         if path.ext == ".py":
-            return StbtCollector(path, parent)
+            if hasattr(StbtCollector, "from_parent"):
+                # pytest >v5.4
+                return StbtCollector.from_parent(parent=parent, fspath=path)  # pylint:disable=no-member
+            else:
+                # Backwards compat
+                # https://docs.pytest.org/en/stable/deprecations.html#node-construction-changed-to-node-from-parent
+                return StbtCollector(path, parent)
         else:
             return None
 
@@ -1235,7 +1241,17 @@ try:
                 for n, line in enumerate(f):
                     m = re.match(r'^def\s+(test_[a-zA-Z0-9_]*)', line)
                     if m:
-                        yield StbtRemoteTest(self, self.name, m.group(1), n + 1)
+                        if hasattr(StbtRemoteTest, "from_parent"):
+                            # pytest >v5.4
+                            srt = StbtRemoteTest.from_parent(  # pylint:disable=no-member
+                                parent=self, filename=self.name,
+                                testname=m.group(1), line_number=n + 1)
+                        else:
+                            # Backwards compat
+                            # https://docs.pytest.org/en/stable/deprecations.html#node-construction-changed-to-node-from-parent
+                            srt = StbtRemoteTest(
+                                self, self.name, m.group(1), n + 1)
+                        yield srt
 
 
     class StbtRemoteTest(pytest.Item):

